lib/Service/OpenProjectAPIService.php

```diff
@@ -1119,13 +1119,21 @@ private function isGroupfolderAppCorrectlySetup():bool {
 		return false;
 	}
 
+	/**
+	 * @return int
+	 */
+	public function getPasswordLength(): int {
+		$passLengthSet = (int) $this->config->getAppValue('password_policy', 'minLength', '');
```

Comment:
should we add 72 here so that the default will be 72 if the value is not set at all.
```suggestion
		$passLengthSet = (int) $this->config->getAppValue('password_policy', 'minLength', 72);
```

Reply:
i have added default to 0 and any ways it will be if less than 72 the function will return lenght 72